chispa/row_comparer.py

```diff
@@ -1,10 +1,27 @@
 from pyspark.sql import Row
+from chispa.number_helpers import isnan, nan_safe_equality
 
 
 def are_rows_equal(r1: Row, r2: Row) -> bool:
     return r1 == r2
 
 
+def are_rows_equal_enhanced(r1: Row, r2: Row, allow_nan_equality: bool) -> bool:
```

Comment:
Do you still need `are_rows_equal()` with this new function in town?

Reply:
Not needed by the dataframe comparison code anymore, but will keep it to maintain backwards compatibility.